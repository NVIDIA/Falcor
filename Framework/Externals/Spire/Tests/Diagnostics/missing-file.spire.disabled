// trying to import a non-existant file

using "does-not-exist.spire"
