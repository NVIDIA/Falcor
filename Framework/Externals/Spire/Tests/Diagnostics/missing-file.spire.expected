result code = -1
standard error = {
Tests/Diagnostics/missing-file.spire(0): error 20001: unexpected end of file, expected ';'
Tests/Diagnostics/missing-file.spire(3): error 2: cannot find file 'does-not-exist.spire'.
}
standard output = {
}
